### ISSUE TITLE
Problem displaying ammo/health (still in the corner)

### ISSUE THREAD

**Issue by Honeycombtroy** [raysfire/rayshud#60]
![20170216212804_1](https://cloud.githubusercontent.com/assets/25817712/23017935/43604356-f490-11e6-9046-81ecafa398ad.jpg)

The health is still in the corner and the ammo is cut off also in the corner, not to mention the bat and bonk meters are where they usually are rather than where the hud should put them. Probably a problem on my part somewhere in the installation but where/how can i fix it?

**Comment by electricman226** [comment]
Probably installed incorrectly. Do you have another hud installed? Could be that you replaced an old rayshud version rather than delete the old one and then paste the new one in.

**Comment by CriticalFlaw** [comment]
It's also possible that you've overwritten stock tf2 files with that of rayshud, causing this effect.
The downloaded HUD folder should be in your custom folder like so:

![custom folder](http://puu.sh/u6c7v/02ec0380ba.png)

if you're still experiencing issues, I'd suggest verifying integrity of game files

**Comment by SiIky** [comment]
What about resolution/aspect ratio? I think that's the most likely cause. The screen shot you attached is 1280x600, is that the resolution of your screen (or the resolution you're using in-game) or did you crop it?

**Comment by electricman226** [comment]
How could he have cropped it, and how could that be the cause?
If he cropped it, it would have cut out he health and ammo, and what could that say about the cause of it?

**Comment by CriticalFlaw** [comment]
The issue is that the game is using default hud assets not rayshud, except for the font. The resolution couldn't be the cause for this.

**Comment by SiIky** [comment]
> How could he have cropped it

GIMP, PS, <insert generic image editing program here>... And who knows if or why he would crop the screenshot?

> and how could that be the cause?

It wouldn't be the cause but it could mislead anyone into thinking the problem is something other than the real problem.

@CriticalFlaw I don't even remember what the stock HUD looks like. To me the HUD on the screenshot looks like raysHUD, just cropped at the edges. And, seriously, 1280x600? I **really** think I've never seen that resolution before. 

According to the [huds.tf page](http://huds.tf/forum/showthread.php?tid=377), RaysHUD supports 16:9, 16:10, 4:3 and 5:4.

```
16:9 = ~1.(7)
16:10 = 1.6
4:3 = ~1.(3)
5:4 = 1.25
1280:600 = ~2.1(3)
```

Sure it doesn't say those are the only supported aspect ratios, but if his aspect ratio is not on the list, there are no guarantees it'll work.

Hence me thinking resolution could be the problem.

**Comment by Honeycombtroy** [comment]
This is the first hud i've ever used and also the first time i've ever put a hud into the folder, so there's no way it's possibly an issue there.
The resolution is no problem, i swapped to multiple different ones and the problem was still there, it must be in the installation somewhere but i have no idea where, I'm new to this whole hud thing and everything

**Comment by SiIky** [comment]
On 17 February 2017 00:11:10 GMT+00:00, Honeycombtroy <notifications@github.com> wrote:
>This is the first hud i've ever used and also the first time i've ever
>put a hud into the folder, so there's no way it's possibly an issue
>there.
>The resolution is no problem, i swapped to multiple different ones and
>the problem was still there, it must be in the installation somewhere
>but i have no idea where, I'm new to this whole hud thing and
>everything

It's alright. If resolution/aspect ratio wasn't the problem maybe it really was installation.

Can you tell us the exact steps you did to install?

Have you tried reinstalling?

And can you post the console output right after startup? If there's something wrong anywhere it should show up on the console.

**Comment by IdleEllis** [comment]
I have the same problem I got

Couldn't find custom font file 'resource/fonts/Cerbetica_bold_medi

thus it's a font/text problem

**Comment by SiIky** [comment]
@IdleEllis That shouldn't be the problem, but to solve that error, on your PC try deleting the lines from 116 to 120 of [this file](https://github.com/raysfire/rayshud/blob/e73daed5c4cac8881e7bff01718c709bc1109adf/resource/ClientScheme.res#L116).

Also see [here](https://github.com/raysfire/rayshud/pull/61).

**Comment by CriticalFlaw** [comment]
This should now be fixed with the latest update (2017.06.07), since it includes the fix by @SiIky
